src/main/java/org/team1540/robot2025/subsystems/arm/ArmIOReal.java

```diff
@@ -0,0 +1,134 @@
+package org.team1540.robot2025.subsystems.arm;
+
+import static org.team1540.robot2025.subsystems.arm.ArmConstants.*;
+
+import com.ctre.phoenix6.BaseStatusSignal;
+import com.ctre.phoenix6.StatusSignal;
+import com.ctre.phoenix6.configs.CANcoderConfiguration;
+import com.ctre.phoenix6.configs.Slot0Configs;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.MotionMagicVoltage;
+import com.ctre.phoenix6.controls.VoltageOut;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.*;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.units.measure.*;
+
+public class ArmIOReal implements ArmIO {
+    private final TalonFX motor = new TalonFX(MOTOR_ID);
+    private final CANcoder cancoder = new CANcoder(CANCODER_ID);
+
+    private final StatusSignal<Angle> position = motor.getPosition();
+    private final StatusSignal<AngularVelocity> velocity = motor.getVelocity();
+    private final StatusSignal<Voltage> appliedVoltage = motor.getMotorVoltage();
+    private final StatusSignal<Current> current = motor.getSupplyCurrent();
+    private final StatusSignal<Temperature> temp = motor.getDeviceTemp();
+    private final StatusSignal<ForwardLimitValue> forwardLimit = motor.getForwardLimit();
+    private final StatusSignal<ReverseLimitValue> reverseLimit = motor.getReverseLimit();
```

Comment:
These can never be true because krakens don't have hardwired limit switches. If we use limit switches (I don't think we will), they'd probably be with a CANdi or RIO

Reply:
This will not trip for the soft limits

https://v6.docs.ctr-electronics.com/en/stable/docs/api-reference/api-usage/actuator-limits.html